### PR TITLE
feat: added line blank space in circle progress

### PR DESCRIPTION
**Pull request by lucas-silveira**
More option of design.

print

### CODE REVIEW

File: src/CircularProgress.js [hunk 1]
```diff
@@ -1,23 +1,35 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { View, ViewPropTypes } from 'react-native';
-import { Svg, Path, G } from 'react-native-svg';
+import React from "react";
+import PropTypes from "prop-types";
+import { View, ViewPropTypes } from "react-native";
+import { Svg, Path, G } from "react-native-svg";
```
Comment (lucas-silveira) [inline src/CircularProgress.js]: I forgot to import Line component from "react-native-svg".

**Comment by aboveyunhai** [comment]
Could you explain more about what this PR does. Sample minimal lines of code that toggles the feat with some pics might be helpful.

**Comment by Parthhyperlink** [comment]
Hello When you are going to merge this i need this as  i am looking for the same thing

**Comment by lucas-silveira** [comment]
> Could you explain more about what this PR does. Sample minimal lines of code that toggles the feat with some pics might be helpful.



> Could you explain more about what this PR does. Sample minimal lines of code that toggles the feat with some pics might be helpful.

Basically, call the component with props: `lineSpaceWidth` and `lineSpaceColor`.

**Pull request closed without merging**